Classes/Custom/DataHandler.php

```diff
@@ -67,8 +67,10 @@ public function processDatamap_postProcessFieldArray( $status, $table, $id, arra
 		}
 
 
+		$prefix = 'aimeos_';
+
 		// Only listen to own plugin data
-		if( !empty( $listType ) && compare( $listType, 'aimeos_', 1 ) === 0 )
+		if( !empty( $listType ) && substr_compare( $listType, $prefix, 0, strlen( $prefix ) ) === 0 )
```

Comment:
`strncmp( $listType, 'aimeos_', 7 )` would be better in this case.

Doesn't it work at all? I thought we need that as soon as the eID dispatcher for the JSON calls is used?